recipe/gen_patch_json.py

```diff
@@ -1232,6 +1232,25 @@ def _gen_new_index(repodata, subdir):
             elif record["version"] == "0.7.14":
                 _replace_pin("python >=2.7", "python >=2.7,<3.10", deps, record)
 
+        if record_name == "conda-build":
+            # Work around vendored auxlib import error; see:
+            #   https://github.com/conda/conda-build/issues/4333
+            #   https://github.com/conda/conda-build/pull/4335
+            #   https://github.com/conda-forge/conda-build-feedstock/pull/169
+            #
+            # Conda 4.11.1 should fix this upstream, so all we really need to
+            # do is avoid 4.11.0 specifically.
```

Comment:
I'd go for a `conda <4.11.0` restriction and personally think it'd be fine to not "fix" it in `conda`since it's rather a fault in `conda-build`.